src/javascript/app/pages/user/account/settings/financial_assessment.js

```diff
@@ -30,20 +30,30 @@ const FinancialAssessment = (() => {
         });
     };
 
+    const displayHighRiskClassification = () => {
+        const status       = State.getResponse('get_account_status.status');
+        const is_high_risk = /high/.test(State.getResponse('get_account_status.risk_classification'));
+
+        $('#high_risk_classification').setVisibility(
+            is_high_risk && Client.isAccountOfType('financial') ?
+                /(financial_assessment|trading_experience)_not_complete/.test(status) :
+                /financial_assessment_not_complete/.test(status)
```

Comment:
this looks like exactly the check in header, but missing the jp client check. should we move it to a common function and call it in both places?